dojo/settings/settings.dist.py

```diff
@@ -1260,6 +1260,7 @@ def saml2_attrib_map_format(dict):
     'KubeHunter Scan': ['title', 'description'],
     'kube-bench Scan': ['title', 'vuln_id_from_tool', 'description'],
     'Threagile risks report': ['title', 'cwe', "severity"],
+    'MSDefender API': ['unique_id_from_tool'],
```

Comment:
If `unique_id_from_tool` is the only field for hashcode, it is better to set a different algo in  `DEDUPLICATION_ALGORITHM_PER_PARSER`